src/Integration/Helpers/RuleSetHelper.cs

```diff
@@ -75,48 +74,6 @@ public static IEnumerable<RuleSetInclude> FindAllIncludesUnderRoot(RuleSet ruleS
                                             });
         }
 
-        /// <summary>
-        /// Return a non-nested include from source to target
-        /// </summary>
-        /// <param name="source">Required</param>
-        /// <param name="target">Required</param>
-        /// <returns><see cref="RuleSetInclude"/> or null if not found</returns>
-        public static RuleSetInclude FindInclude(RuleSet source, RuleSet target)
-        {
-            if (source == null)
-            {
-                throw new ArgumentNullException(nameof(source));
-            }
-
-            if (target == null)
-            {
-                throw new ArgumentNullException(nameof(target));
-            }
-
-            string relativeTargetFilePath = PathHelper.CalculateRelativePath(source.FilePath, target.FilePath);
-
-            var matchingRuleSetIncludes = source.RuleSetIncludes.Where(i =>
-                StringComparer.OrdinalIgnoreCase.Equals(i.FilePath, relativeTargetFilePath)
```

Comment:
The old code didn't handle complex relative paths or alternative path separators.
I've fixed this is in the new code by converting relative paths to absolute paths in a canonical form.